src/Sunburst.ts

```diff
@@ -559,14 +833,6 @@ export class Sunburst implements IVisual {
         data.total += newDataPointNode.value;
         newDataPointNode.children = [];
 
-        if (level === 1 && originParentNode.children.length > 0) {
```

Comment:
Why are you removing these lines?

Reply:
Because they don't do anything (their variables are never used) so the linter will complain if they are not removed